plugins/modules/execute_lambda.py

```diff
@@ -260,8 +260,10 @@ def main():
 
```

Comment:
I'm not sure if check_mode works correctly on lambda_execute, but I assume it's outside the scope of this PR.